src/locales/en.ts

```diff
@@ -43,15 +43,7 @@ export default {
       phoneNumber: "Phone Number",
     },
     confirm: "Send",
-    error: {
-      db: "Something went wrong! Try again or send an email instead to",
-      email:
-        "Your registration has been received, but we could not send a confirmation email to you.",
-      continue: "Continue anyway",
-    },
-    organizationNumberLength: "The organization number must be 10 digits",
-    organizationNumberChecksum:
-      "Invalid check digit, check that you have entered the number correctly",
+    ignoreError: "Continue anyway",
```

Comment:
Why are we ignoring the error now? Is it because of trpc?

Reply:
I have just moved the errors to its own object and renamed some keys